_includes/alt_header.html

```diff
@@ -0,0 +1,40 @@
+<nav id="navbar" class="navbar navbar-default navbar-fixed-top">
+  <div class="container">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <div class="navbar-header">
+      <a class="navbar-brand" href="/#intro">
+        <img alt="Open Systems Pharmacology" class="logo" src="img/osp_logo.png">
+      </a>
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#osp-navbar-collapse">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+    </div>
+
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <div class="collapse navbar-collapse" id="osp-navbar-collapse">
+      <ul class="nav navbar-nav navbar-right">
+        <li class="hidden">
+          <a href="/#intro"></a>
+        </li>
+        <li><a href="/#essentials">Essentials</a></li>
+        <li><a href="/#news">News &amp; Release</a></li>
```

Comment:
This header is the same as the SPA header but I need this /#news instead of #news because the link is to an anchor on another page

Reply:
I don't undertand this.
What does /#news does? Which page is it going to go to? the main page?
would this work also to do the same on the other header

Reply:
goes to the main page of the site and then the anchor `#news`. You can't use it on the main page because it forces a reload of the page as well as navigation to the anchor.